package.json

```diff
@@ -10,20 +10,41 @@
   "devDependencies": {
     "@swc/core": "^1.2.108",
     "@swc/jest": "^0.2.5",
-    "@types/node": "^16.10.2",
     "esbuild": "^0.13.14",
     "esbuild-jest": "^0.5.0",
+    "@types/node": "^16.11.12",
+    "@typescript-eslint/eslint-plugin": "^5.6.0",
+    "@typescript-eslint/parser": "^5.6.0",
+    "eslint": "^8.4.1",
+    "eslint-config-prettier": "^8.3.0",
+    "eslint-config-standard": "^16.0.3",
+    "eslint-plugin-import": "^2.25.3",
+    "eslint-plugin-node": "^11.1.0",
+    "eslint-plugin-prettier": "^4.0.0",
+    "eslint-plugin-promise": "^5.2.0",
+    "eslint-plugin-react": "^7.27.1",
+    "husky": "^7.0.4",
     "lerna": "^4.0.0",
+    "lint-staged": "^12.1.2",
+    "prettier": "^2.5.1",
     "rimraf": "^3.0.2",
+    "typescript": "^4.5.2",
     "window-size": "^1.1.1"
   },
   "scripts": {
     "build": "node ./scripts/build-package.js",
     "test": "jest --runInBand",
-    "prepare": "lerna run prepare"
+    "prepare": "husky install && lerna run prepare",
```

Comment:
To confirm, the installation procedure currently is:

yarn (Install deps)
yarn prepare (Install husky, prepare each folder)
yarn build (From monorepo root)

Testing
yarn test (From monorepo root)

Reply:
Should yarn prepare run the build scripts in 
1. React apps: ezbackend-db-ui, ezcrypto and packages/docs 
2. Each individual package

That way we make the build process easier for the end user

Also can we reflect the initialization process in CONTRIBUTING.md